services/docker-compose.yml

```diff
@@ -173,6 +173,7 @@ services:
     profiles:
       - all
       - service
+      - main
```

Comment:
Indexing service is now part of the `main` profile.  This is required because we are now using Elasticsearch for results.